stubs/protobuf/google/protobuf/internal/extension_dict.pyi

```diff
@@ -9,28 +9,14 @@ _ExtenderMessageT = TypeVar("_ExtenderMessageT", bound=Message)
 class _ExtensionFieldDescriptor(FieldDescriptor, Generic[_ContainerMessageT, _ExtenderMessageT]): ...
 
 class _ExtensionDict(Generic[_ContainerMessageT]):
-    def __init__(self, extended_message: _ContainerMessageT) -> None: ...
-    # Dummy fallback overloads with FieldDescriptor are for backward compatibility with
-    # mypy-protobuf <= 1.23. We can drop them a few months after 1.24 releases.
-    @overload
+    def __init__(self, extended_message: _ContainerMessageT): ...
```

Comment:
this made stubtest happy

Reply:
matching https://github.com/protocolbuffers/protobuf/blob/master/python/google/protobuf/internal/extension_dict.py#L71

Reply:
This should not be needed. Can you please re-add `-> None` so we can have a look at the failure in CI?

Reply:
ah this was absolutely a mistake on my part. I had
1) Accidentally deleted the __init__
2) Saw a CI failure because it was missing
3) Readded it manually, but poorly.

Undoing my error!